src/Domain/Model/DynamoDbModel.php

```diff
@@ -183,6 +183,10 @@ public function save(array $options = [])
             $this->fireModelEvent('creating');
         }
 
+        if ($this->usesTimestamps()) {
```

Comment:
Lumen's `Eloquent/Model` also set's the timestamps before persisting to the storage. I kept the logic as is. 

Can you give me a note why we shouldn't persist the timestamps with `$this->client->putItem();`?